flow360/component/flow360_params/solvers.py

```diff
@@ -514,6 +516,24 @@ class TransitionModelSolver(GenericFlowSolverSettings):
             displayed="Linear solver config",
         )
 
+    # pylint: disable=missing-class-docstring,too-few-public-methods
+    class Config(Flow360BaseModel.Config):
+        conflicting_fields = [Conflicts(field1="N_crit", field2="turbulence_intensity_percent")]
```

Comment:
Let's put `Config` as the last item in class definition.

Reply:
Done.

BTW I found that setting `N_crit` before setting `turbulence_intensity_percent` to None (in the `to_solver `function) reports validation error? Seems that it constantly check whether the `Config` is satisfied.